Makefile

```
@@ -244,6 +245,23 @@ OPM = $(shell which opm)
 endif
 endif
 
+.PHONY: operator-sdk
+OPERATOR_SDK = $(shell pwd)/bin/operator-sdk
+operator-sdk: ## Download operator-sdk locally if necessary.
+ifeq (,$(wildcard $(OPERATOR_SDK)))
+ifeq (,$(shell which operator-sdk 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPERATOR_SDK)) ;\
+	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+	curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.15.0/operator-sdk_${OS}_${ARCH} ;\
```

On my environment (Fedora 36, GNU Make 4.3, GNU bash, version 5.2.15) `${OS}` and `${ARCH}` are empty, I can fix this issue only moving OS and ARCH:
```suggestion
OS=$(shell go env GOOS)
ARCH=$(shell go env GOARCH)
OPERATOR_SDK = $(shell pwd)/bin/operator-sdk
operator-sdk: ## Download operator-sdk locally if necessary.
ifeq (,$(wildcard $(OPERATOR_SDK)))
ifeq (,$(shell which operator-sdk 2>/dev/null))
	@{ \
	set -e ;\
	mkdir -p $(dir $(OPERATOR_SDK)) ;\
	curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.15.0/operator-sdk_${OS}_${ARCH} ;\
```

---

great, that may explain why there was a failure in the CI, https://github.com/artemiscloud/activemq-artemis-operator/actions/runs/4253007736/jobs/7397300062#step:5:268

---

that OS/ARCH pattern is used in the OPM target, so it can also benefit :-)